Packs/HostIo/Integrations/HostIo/hostio.py

```diff
@@ -0,0 +1,229 @@
+from typing import List, Optional, Union
+import urllib3
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def get_domain_data(self, domain: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'full/{domain}',
+            params={}
+        )
+
+    def get_search_data(self, field: str, value: str, limit: int) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'domains/{field}/{value}',
+            params={'limit': limit}
+        )
+
+    def test_module(self):
+        return self._http_request(
+            method='GET',
+            url_suffix='domains/ip/8.8.8.8',
+            params={}
+        )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def parse_domain_date(domain_date: Union[List[str], str], date_format: str = '%Y-%m-%dT%H:%M:%S.000Z') -> Optional[str]:
```

Comment:
You are converting the date format to be the same as the result's date format
```suggestion
def parse_domain_date(domain_date: Union[List[str], str], date_format: str = DATE_FORMAT) -> Optional[str]:
```

Reply:
done